### PR TITLE
PR for BXMSDOC-7493: Single-source and add "Data source management" chapter in the upstream document

### PR DESCRIPTION
**Pull request by kaldesai**
Associated JIRA: https://issues.redhat.com/browse/BXMSDOC-7493

**Enterprise document previews:**
- [RHPAM-II. Configuring Business Central settings and properties](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-7493-RHPAM-DSM-BC/#data-source-management-con_configuring-central)
- [RHDM-II. Configuring Business Central settings and properties](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-7493-RHDM-DataSource/#data-source-management-con_configuring-central)

**Doc impact for enterprise documents:**
- Check section **28. Data source and database driver management** in RHPAM doc
- Check section **22. Data source and database driver management** in RHDM doc (Same as RHPAM)

**Community document previews:**
- [Drools document](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-Drools-DataSource/#data-source-management-con_kie-apis)
- [jBPM document](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-jBPM-DataSource/#data-source-management-con_business-processes)

**Doc impact for community documents:**
- Check **15.8.9. Data source and database driver management** in Drools doc
- Check **13.8.9. Data source and database driver management** in jBPM doc (Same as Drools)

Can you please review and verify the community doc and share your feedback?

### CODE REVIEW

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/adding-database-driver-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+= Adding a database driver
+
+You can add a new database driver to {CENTRAL}.
+
+.Procedure
+. In {CENTRAL}, select the *Admin* icon in the upper-right corner of the screen and select *Data Sources*.
+. In the *DataSource Explorer* pane, click *Add Driver*.
++
+The *New driver* window opens.
+. In the *New driver* window, enter the *Name*, *Driver Class Name*, *Group Id*, *Artifact Id*, and *Version* of the database driver.
```
Comment (emmurphy1) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/adding-database-driver-proc.adoc:12]: Remove the bolding and capitals. You are not enter the UI elements, you are entering values.
 In the *New driver* window, enter the name, driver class name, group ID, artifact ID, and version* of the database  driver. Also, ID is always capital (ID).

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/data-source-management-con.adoc [hunk 2]
```diff
@@ -1,5 +1,5 @@
-[id='managing-business-central-data-sources-con']
-= Data source management
+[id='data-source-management-con_{context}']
+= Data source and database driver management
 
 {CENTRAL} provides data source management features that enable you to define data sources for accessing a database. These data sources are then used by other {CENTRAL} components such as the data sets. A database driver enables communication between a data source and the targeted database.
```
Comment (emmurphy1) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/data-source-management-con.adoc:4]: Remove space at start of line.

Comment (emmurphy1) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/data-source-management-con.adoc:4]: such as data sets.

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/adding-database-driver-proc.adoc (hunk 1 above)
Comment (kaldesai) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/adding-database-driver-proc.adoc:12]: Okay, noted and updated

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/data-source-management-con.adoc (hunk 2 above)
Comment (kaldesai) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/data-source-management-con.adoc:4]: I don't see any extra space in the content. Please correct me if I am verifying something else. :) 

image

Comment (kaldesai) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/data-source-management-con.adoc:4]: Updated.

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/adding-database-driver-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+= Adding a database driver
+
+You can add a new database driver to {CENTRAL}.
+
+.Procedure
+. In {CENTRAL}, select the *Admin* icon in the upper-right corner of the screen and select *Data Sources*.
+. In the *DataSource Explorer* pane, click *Add Driver*.
++
+The *New driver* window opens.
+. In the *New driver* window, enter the name, driver class name, group ID, artifact ID, and version of the database driver.
```
Comment (barboras7) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/adding-database-driver-proc.adoc:12]:
```suggestion
. In the *New driver* window, enter the *Name, Driver Class Name, Group Id, Artifact Id*, and *Version* of the database driver.
```

Comment (barboras7) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/adding-database-driver-proc.adoc:12]: I'd suggest to follow the same pattern as in other sections

Comment (kaldesai) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSourceManagement/adding-database-driver-proc.adoc:12]: Okay, I will update the content accordingly.

**Review by barboras7 — APPROVED** [review]

**Pull request merged**